libs/filabridge/include/filament/MaterialEnums.h

```diff
@@ -27,7 +27,7 @@
 namespace filament {
 
 // update this when a new version of filament wouldn't work with older materials
-static constexpr size_t MATERIAL_VERSION = 26;
+static constexpr size_t MATERIAL_VERSION = 27;
```

Comment:
@bejado do we need to up the material version for each breaking change, or only when we branch the rc?

Reply:
Typically we update the material version when the breaking change is made, which is how I know to bump the middle version. Either way is fine though